plugins/clnrest/utilities/rpc_routes.py

```diff
@@ -23,7 +23,7 @@ def get(self):
             return response
 
         except Exception as err:
-            plugin.log(f"Error: {err}", "error")
+            plugin.log(f"Error: {err}", "debug")
```

Comment:
if you change this, is will not show up inside the core lightning logs if the INFO level is set

Reply:
Ok, so maybe we should change from `debug` to `info`.

Reply:
I added a commit such that those logs are now at INFO level.